src/Telegram/Bot/API/Types.hs

```diff
@@ -165,11 +165,13 @@ data MessageEntityType
   | MessageEntityEmail
   | MessageEntityBold
   | MessageEntityItalic
+  | MessageEntityUnderline
+  | MessageEntityStrikethrough
```

Comment:
Please provide URL on Telegram spec, see `MessageEntityCashtag` for example.

Reply:
Sure, done